Library/LibrarianAdd.aspx.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Library.Data;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace Library
+{
+    public partial class LibrarianAdd : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        protected void Save_Click(object sender, EventArgs e)
+        {
+            int employeeNumber = int.Parse(EmployeeNumber.Text);
+            int libary_ID = int.Parse(LibraryList.SelectedValue);
```

Comment:
Variable naming nitpick... I'd name this variable `libraryId` or `libraryID` (avoiding the use of the underscore).